.travis.yml

```diff
@@ -1,16 +1,25 @@
 language: cpp
-os:
-  - linux
 sudo: required
 dist: trusty
-compiler:
-  - g++
-  - clang
 before_install:
-  - sudo add-apt-repository --yes ppa:beineri/opt-qt551-trusty
-  - sudo apt-get update -qq
-  - sudo apt-get install libibus-1.0-dev qt55base qt55declarative
-  - source /opt/qt55/bin/qt55-env.sh
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then
```

Comment:
Better to have it in a script and run the script from travis ci. Then its much cleaner

Reply:
Thanks for the tip! I will try it next time when I work on this PR again.